apps/files/src/components/FileEntry.vue

```diff
@@ -83,7 +83,7 @@
 			class="files-list__row-mtime"
 			data-cy-files-list-row-mtime
 			@click="openDetailsIfAvailable">
-			<span>{{ mtime }}</span>
+			<NcDateTime :timestamp="mDateTime"/>
```

Comment:
I would suggest to remove the seconds with this:

```suggestion
			<NcDateTime :timestamp="mDateTime" :ignore-seconds="true" />
```